src/Class/Hydrator/LayoutDefinitionHydrator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Class\Hydrator;
+
+use Pimcore\Bundle\StudioBackendBundle\Class\Schema\LayoutDefinition;
+use Pimcore\Model\DataObject\Fieldcollection\Definition as FieldCollectionDefinition;
+
+/**
+ * @internal
+ */
+final class LayoutDefinitionHydrator implements LayoutDefinitionHydratorInterface
+{
+    public function hydrate(FieldCollectionDefinition $data): LayoutDefinition
```

Comment:
Should we maybe make this a little more specific? Either use `Field Collection namespace` or if we will share the hydrator, then rename the method to `hydrateFieldCollection` or similiar? WDYT?

Reply:
I will but it into a Field Collection namespace like all the other Service. 

Good Point thx :)

Reply:
Done here [e0011ca](https://github.com/pimcore/studio-backend-bundle/pull/581/commits/e0011cae0ed4aa168eaaa7a0bd2495b54471abf3)